pom.xml

```diff
@@ -204,31 +204,6 @@
                     </webApp>
                 </configuration>
             </plugin>
-            <plugin>
-                <artifactId>maven-clean-plugin</artifactId>
-                <version>3.1.0</version>
-                <configuration>
-                    <filesets>
-                        <fileset>
-                            <directory>.</directory>
-                            <includes>
-                                <include>package.json</include>
-                                <include>package-lock.json</include>
-                                <include>webpack.config.js</include>
-                                <include>webpack.generated.js</include>
-                            </includes>
-                            <followSymlinks>false</followSymlinks>
-                        </fileset>
-                        <fileset>
-                            <directory>./node_modules</directory>
-                            <includes>
-                                <include>**/**</include>
-                            </includes>
-                            <followSymlinks>false</followSymlinks>
-                        </fileset>
-                    </filesets>
-                </configuration>
-            </plugin>
```

Comment:
Any reason why this is removed? It's actually quite useful as vaadin regularly breaks itself between updates if no vaadin:clean-frontend is performed. I would actually go as far as adding the new tsconfig.json to the list of files to be removed when cleaning the project. Either way, this change doesn't seem to be related to the PR title itself.

Reply:
I removed it to speed up the build process. I dont see the value in removing `node_modules` everytime when you are building the project. 

When you are building the project without `package*.json` it raises a warning:
```
[INFO] --- vaadin-maven-plugin:23.2.0:build-frontend (default) @ apexcharts ---
[INFO] Reflections took 203 ms to scan 76 urls, producing 2573 keys and 14623 values
[INFO] Scanning classes to find frontend configurations and dependencies...
[INFO] Visited 590 classes. Took 556 ms.
[INFO] Visited 104 classes. Took 41 ms.
[INFO] Unable to determine version information. No vaadin-core-versions.json found
[INFO] Running `npm install` to resolve and optionally download frontend dependencies. This may take a moment, please stand by...
[INFO] using 'C:\Program Files\nodejs\npm.cmd --no-update-notifier --no-audit --scripts-prepend-node-path=true --ignore-scripts install' for frontend package installation
[WARNING] package-lock.json is missing from this project. This may cause the npm package installation to take several minutes. It is recommended to keep the package-lock.json file persistently in your project. Please stand by...
[INFO] Frontend dependencies resolved successfully.
[INFO] Copying frontend resources from jar files ...
[INFO] Visited 77 resources. Took 218 ms.
[INFO] Copying project local frontend resources.
[INFO] Copying frontend directory completed.
[INFO] Reflections took 99 ms to scan 76 urls, producing 2573 keys and 14623 values
[INFO] Running Vite ...
[INFO] Build frontend completed in 20487 ms.
```

What do you think?

Reply:
I have a similar maven-clean-plugin include in my own vaadin projects, and it only runs if you execute 'maven clean'. If you just compile the project using 'maven compile' or 'maven build' , the node_modules folder stays as it is. Cleaning up the node_modules folder every build isn't right, I agree with that. The point of this plugin definition is that if the build is somehow broken, running 'maven clean install' will also remove all potentially broken generated files and npm packages, before building again.
Either way, I think this change belongs in a different PR, or at least a separate commit.

Reply:
Moved to #143